src/Mvc/Mvc.ViewFeatures/src/RazorComponents/ComponentRenderer.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures
+{
+    internal class ComponentRenderer : IComponentRenderer
```

Comment:
Are there any changes to what was there in the HtmlHelper?

Reply:
Nope, this was pretty much copied from the old code as-is.